src/ai/backend/client/func/vfolder.py

```diff
@@ -394,6 +433,15 @@ async def mkdir(
         async with rqst.fetch() as resp:
             return await resp.text()
 
+    @api_function
+    async def mkdir(
+        self,
+        path: Union[str, Path],
+        parents: Optional[bool] = False,
+        exist_ok: Optional[bool] = False,
+    ):
```

Comment:
Following the type annotation you've added:
```suggestion
    ) -> str:
```